src/threading/thread.h

```diff
@@ -158,9 +203,9 @@ class Thread {
 	Mutex m_mutex;
 
 #if !USE_CPP11_THREADS
```

Comment:
This should be #if**n**def USE_CPP11_THREADS, or (better) combined with the #if USE_CPP11_THREADS on line 220